src/npctalk.cpp

```diff
@@ -654,6 +659,10 @@ std::string dialogue::dynamic_line( const talk_topic &the_topic ) const
                    _( "&You are deaf and can't talk. When you don't respond, %s becomes angry!" ),
                    beta->name.c_str() );
     }
+    if( topic == "TALK_SEDATED" ) {
+        return string_format( _( "%s is sedated and can't wake up until the sedation wears off" ),
```

Comment:
Suggest changing to `%s is sedated and cannot be moved or woken up until the [medication/sedation] wears off.`

Reply:
Fixed.